README.md

```diff
@@ -901,6 +901,115 @@ _You can enable the following settings in Xcode by running [this script](resourc
   }
   ```
 
+* <a id='conditional-assignment'></a>(<a href='#conditional-assignment'>link</a>) **When initializing a new property with the result of a conditional statement (e.g. an `if` or `switch` statement), use a single if/switch expression where possible** rather than defining an uninitialized property and initializing it on every branch of the following conditional statement. [![SwiftFormat: conditionalAssignment](https://img.shields.io/badge/SwiftFormat-conditionalAssignment-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#conditionalAssignment)
+
+  <details>
+
+  #### Why?
+
+  There are several benefits to using an if/switch expression over simply performing assignment on each branch of the following conditional statement:
+  1. In most cases, you no longer need to explicitly write a type annotation for the variable that is being assigned to.
```

Comment:
I noticed that we add a period after each numbered item here, but we don't have a period in https://github.com/airbnb/swift/pull/253. Let's be consistent, one way or the other.